pandapower/create.py

```diff
@@ -3975,15 +3937,12 @@ def create_transformer3w_from_parameters(
         vkr_mv_percent, vkr_lv_percent, pfe_kw, i0_percent,
         shift_mv_degree=0., shift_lv_degree=0., tap_side=None,
         tap_step_percent=nan, tap_step_degree=nan, tap_pos=nan,
-        tap_neutral=nan, tap_max=nan,
+        tap_neutral=nan, tap_max=nan, tap_phase_shifter_type=nan,
```

Comment:
Default to something explicit, like None?

Reply:
This is merely following the same logic as all the other trafo parameters. The default value is later on defined as pd.NA if no integer value is provided. If we change this parameter from int to str, this needs to be revisited.